### DISCUSSION TITLE
How can I have a theme-sensative SVG?

### DISCUSSION THREAD

**Discussion by benjitusk** [squidfunk/mkdocs-material#5884] [Q&A]
I have 2 SVG files on disk, and I want to display the appropriate one depending on the theme. I read the docs where it says to append #only-light and #only-dark to the file path, but it seems to be breaking (when I use `mkdocs serve`), presumably because it thinks #only-dark is part of the actual file path. Has anyone else managed with this?

**Comment by squidfunk** [comment]
A [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) goes a long way ☺️ It's much easier to help you if we have a self contained example that we can run.
